frontend/src/components/form/index.tsx

```diff
@@ -29,19 +33,88 @@ Form.Title = function FormTitle({ children, ...restProps }: any) {
   return <Title {...restProps}>{children}</Title>
 }
 
-Form.Input = function FormInput({ setInput, ...restProps }: any) {
+Form.Input = function FormInput({ handleInputChange, ...restProps }: any) {
   return (
     <Input
       marginY={[1, 2]}
       padding={[1, 2]}
       {...restProps}
-      onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-        setInput(event.target.value)
-      }
+      onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+        console.log(event.target.value)
+        handleInputChange(event)
+      }}
     />
   )
 }
 
+export const InputField = ({
+  validation,
+  name,
+  register,
+  errors,
+  ...restProps
+}: any) => {
+  const error = errors[name] ? (
+    <Error>{errors[name].message}</Error>
+  ) : (
+    <Error>*</Error>
+  )
+  return (
+    <>
+      <Input
+        marginY={[1, 2]}
+        padding={[1, 2]}
+        name={name}
+        type={validation.type}
+        {...register(name, validation)}
+        {...restProps}
+      />
+      {error}
+    </>
+  )
+}
+
+export const PasswordField = ({
```

Comment:
This should be in a separate file as well.

Reply:
Napravila sam 1 input i da u zavisnosti od tipa moze da mu se ubaci visiblityTiggler.
Ali sam ostavila pod komentarima InputPassword, da vidi neko dal je bolje da bude odvojeno ili ne, pa da resim.